include/zephyr/mgmt/hawkbit/config.h

```diff
@@ -45,6 +45,7 @@ struct hawkbit_runtime_config {
  * @param config Configuration settings to set.
  * @retval 0 on success.
  * @retval -EAGAIN if probe is currently running.
+ * @retval -EINVAL if config->server_addr is too long.
```

Comment:
should be a more generic description, as we could in the future also check config->auth_token f.e.

Reply:
I changed it to be more generic in most recent force push